pkg/resource/aws/aws_route53_zone_test.go

```diff
@@ -12,11 +12,19 @@ func TestAwsRoute53Zone_String(t *testing.T) {
 		zone AwsRoute53Zone
 		want string
 	}{
-		{name: "",
+		{name: "test route53 zone stringer with name and id",
 			zone: AwsRoute53Zone{
 				Name: aws.String("example.com"),
+				Id:   "01",
 			},
-			want: "example.com",
+			want: "example.com (01)",
+		},
+		{name: "test route53 zone stringer without name",
```

Comment:
Interesting test case, but in this case we should only display the id

```
want: "02"
```

Reply:
I'm not sure to understand why you want only the Id when the Name is available (and required) for this resource.
But at least I can display only the Id when then Name is nil.

Reply:
Done.